config/config.default.js

```diff
@@ -17,9 +17,11 @@ module.exports = () => {
    * @property {Object} xssProtection - 是否开启IE8的XSS Filter，默认开启。
    * @property {Object} csp - csp安全配置。
    * @property {Array} domainWhiteList - 安全跳转白名单
+   * @property {Array} protocolWhiteList - 协议白名单
    */
   exports.security = {
     domainWhiteList: [],
+    protocolWhiteList: [],
```

Comment:
Surely consistent config style would be our choice.

There are two styles:

- `config.security.domainWhiteList & config.security.protocolWhiteList`
- `config.helper.shtml.domainWhiteList & config.helper.shtml.protocolWhiteList`

Both are okey with me, but previous one brings a break change, so I prefer the latter one.